sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2433,6 +2433,26 @@ pi_result cuda_piextMemCreateWithNativeHandle(pi_native_handle nativeHandle,
   return {};
 }
 
+/// Created a PI image mem object from a CUDA image mem handle.
+/// TODO: Implement this.
+/// NOTE: The created PI object takes ownership of the native handle.
+///
+/// \param[in] nativeHandle The native handle to create PI mem object from.
+/// \param[in] context The PI context of the memory allocation.
+/// \param[in] ownNativeHandle Indicates if we own the native memory handle or
+/// it came from interop that asked to not transfer the ownership to SYCL RT.
+/// \param[out] mem Set to the PI mem object created from native handle.
+///
+/// \return TBD
+pi_result cuda_piextImgCreateWithNativeHandle(
+    pi_native_handle nativeHandle, pi_context context, bool ownNativeHandle,
+    const pi_image_format *ImageFormat, const pi_image_desc *ImageDesc,
+    pi_mem *Img) {
```

Comment:
Should the unused arguments be silenced?

Reply:
I modelled this after `cuda_piextMemCreateWithNativeHandle` which likewise does not silence unusued args.  I'm happy to do so, just wasn't sure what practice is being followed for the CUDA Plugin.